src/app/app.module.ts

```diff
@@ -37,6 +38,12 @@ import { HeaderComponent } from './shared/layout';
 import { markedOptionsFactory } from './shared/lib/marked';
 import { SharedModule } from './shared/shared.module';
 
+export const customTooltipDefaults: MatTooltipDefaultOptions = {
+  showDelay: 1000,
+  hideDelay: 0,
+  touchendHideDelay: 0
+};
+
```

Comment:
Similarly, should you be changing the behaviour of all mat tooltips? Other tooltips throughout the application have unintentionally been forced to follow the new defaults. Perhaps it would be better if absolutely necessary to set the custom options on the respective custom tooltips as required. On another note, is there a reason for these specific overrides of the defaults?

Reply:
Hi, decided to go with this at first since i didnt notice any other tooltips on the application, but I do understand why this practice may not be ideal. will change it out or minimally narrow the scope at least